docs/sources/reference/http_api.md

```diff
@@ -70,13 +70,15 @@ Creates a new dashboard or updates an existing dashboard.
         "schemaVersion": 6,
         "version": 0
       },
-      "overwrite": false
+      "overwrite": false,
+      "userId:": 3
```

Comment:
why is this in the write request? The UserId is taken from request user/auth context.